src/app/workflow/executions/executions-tab.component.html

```diff
@@ -13,7 +13,7 @@
       <mat-label>Select a Platform</mat-label>
       <mat-select data-cy="metrics-partner-dropdown" [value]="currentPartner" (selectionChange)="matSelectChange($event)">
         <mat-option data-cy="metrics-partner-options" [value]="partner" *ngFor="let partner of partners">
-          {{ partner }}
+          {{ getPartnerDisplayName(partner) }}
```

Comment:
You don't need this function. Instead you can call the pipe like this: `{{ partner | platformPartner }}`